src/diagnostics/Diagnostic.H

```diff
@@ -26,25 +26,25 @@ public:
     void AllocData (int lev, const amrex::Box& bx, int nfields, amrex::Geometry const& geom);
 
     /** \brief return the main diagnostics multifab */
-    amrex::Vector<amrex::FArrayBox>& getF () { return m_F; };
+    amrex::Vector<amrex::FArrayBox>& getF () { return m_F; }
 
     /** \brief return the main diagnostics multifab
      *
      * \param[in] lev MR level
      */
-    amrex::FArrayBox& getF (int lev) {return m_F[lev];};
+    amrex::FArrayBox& getF (int lev) { return m_F[lev]; }
```

Comment:
We sometimes have a space like ` { return m_F[lev]; }` and sometimes we don't  `{return m_F[lev];}`. Could we make this consistent?
Maybe we should add this and where to set spaces etc in loops and if statements to our style guide.

Nonetheless, I will merge this PR now, that is a separate issue.